index.html

```diff
@@ -0,0 +1,172 @@
+                <div class="exibition">
+                    <div class="redlogo-container">
+                        <img src="assets/images/teacher.png" class="image-1" alt="">
+                    </div>
+                    <h3>Leacture</h3>
+                    <p>December 1st kicks off AIDS Awareness Month with World AIDS Day. Leading up to the month of December was the 40th Anniversary</p>
+                </div>
+                <div class="exibition">
+                   <div class="redlogo-container">
+                       <img src="assets/images/darkboard.png" alt="" class="image-1">
+                   </div>
+                    <h3>CC Exhibition</h3>
+                    <p>With HIV related products and services on display in interactive 3D booths; chat functions enabling real-time dialogue with exhibitors</p>
+                </div>
+                <div class="exibition">
+                    <div class="redlogo-container">
+                        <img src="assets/images/chat.png" class="image-1" alt="">
+                    </div>
+                    <h3>Forum</h3>
+                    <p>The Forum HIV Prevention Trial Design Project convenes a series of public meetings.</p>
+                </div>
+                <div class="workshop">
+                  <div class="redlogo-container">
+                      <img src="assets/images/whiteboard.png" alt="">
+                  </div>
+                    <h3>Workshop</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="workshop">
+                    <div class="redlogo-container">
+                        <img src="assets/images/roulette.png" class="image-1" alt="">
+                    </div>
+                    <h3>CC Ignite</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="global">
+                    <h4 class="bar">Join HIV Global Summit 2022</h4>
+                </div>
+            </div>
+        </div>
+        <div   class="program">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </div>
+</section>
+<div id="speaker-list"></div>
+<h2 class="speaker-list">Featured Speakers</h2>
+<div class="center-hor">
+    <div class="horizontal-line"></div>
+</div>
+<div id="speaker-lists"></div>
+<footer>
+    <div class="footer">
+        <h2>Partner</h2>
+        <div class="borders1">
+            <div class="borders"></div>
+        </div>
+        <div class="sub-section">
+            <div class="footer-section">
+                <ul class="lists">
+                    <li>mozilla</li>
+                    <li>Google</li>
+                    <li>NAVER</li>
+                    <li class="link-privacy">daumkakao</li>
+                    <li>airbnb</li>
+                </ul>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly check the placement of your li element they are at the footer section, and place them in the partner section

<img width="776" alt="image" src="https://user-images.githubusercontent.com/61408860/169537663-c77cfa80-2110-4148-b701-2d4062008ea9.png">